chromium_src/third_party/blink/renderer/modules/canvas/canvas2d/base_rendering_context_2d.cc

```diff
@@ -13,7 +13,7 @@
 namespace {
 
 bool IsGoogleMaps(const blink::KURL& url) {
```

Comment:
any host that contains google is a google maps url? Need to check git blame for this

Reply:
Looking on the original PR it seems so https://github.com/brave/brave-core/pull/23416/files

Reply:
I think it's saying any URL where `host == google` and `path == /maps` or `/maps/`